index.html

```diff
@@ -141,7 +141,17 @@ <h2 class="contributors-head">Contributors</h2>
       <p>This project was made possible by many contributors, including the following GitHub users:</p>
 
       <div class="well contributors"><div id="spinner-icon" style="text-align: center;"><i class="fa fa-spinner fa-spin"></i></div><p class="usernames"></p><hr /><p class="avatars"></p></div>
-
+	
+	  <div class="row" id="connectwithus">
+		<h2 class="text-center d-block">Connect with us</h2>
+		<p>You can connect with us on any of these platforms:</pc>
```

Comment:
```suggestion
		<p>You can connect with us on any of these platforms:</p>
```

Reply:
@gauravano  we should html validation to grunt which will automatically detect this type of error

Reply:
Yes, we can surely do this! Nice idea